.github/workflows/build-fb-image.yaml

```diff
@@ -57,63 +48,66 @@ jobs:
             echo "${VERSION} tag does not exist, assuming we're building a new release!"
             IMAGE_BASE_TAG="${VERSION}"
           fi
-  
-          echo "IMAGE_BASE_TAG=$IMAGE_BASE_TAG" >> $GITHUB_OUTPUT
-          echo "IMAGE_MAJOR_MINOR=$MAJOR_MINOR" >> $GITHUB_OUTPUT
+
+          echo "version=$VERSION" >> $GITHUB_OUTPUT
+          echo "image-base-tag=$IMAGE_BASE_TAG" >> $GITHUB_OUTPUT
+          echo "image-major-minor=$MAJOR_MINOR" >> $GITHUB_OUTPUT
```

Comment:
Follow the github actions convention for naming outputs. E.g. see any action outputs. Like the setup-go go-version output or any other popular actions.

This helps also distinguishing the environment variables from the outputs.